web-common/src/features/dashboards/stores/DashboardStateProvider.svelte

```diff
@@ -11,8 +11,9 @@
   import { EntityStatus } from "../../entity-management/types";
 
   export let metricViewName: string;
+  export let additionalKey = ""; // used in cloud to add org and project
```

Comment:
I think it'd be a cleaner interface if the `org` and `project` were the optional props. Then the key could be composed within the `DashboardStateProvider` component and not be leaked outside of it.